static/js/tools/map/chart_loader.tsx

```diff
@@ -194,7 +198,7 @@ export function ChartLoader(): JSX.Element {
 
   if (chartData === undefined) {
     return null;
-  } else if (_.isEmpty(chartData.geoJsonData)) {
+  } else if (_.isEmpty(geoJson)) {
```

Comment:
should we add a way to differentiate between no geojson available vs geojson still loading?

Reply:
Good point! Right now chartData is defined only when there is geoJson data, so it will always exit in the if statement above. As we do more refactor, will need a different state for this.